build.gradle.kts

```diff
@@ -32,24 +24,6 @@ repositories {
     maven { url = uri("https://packages.jetbrains.team/maven/p/ij/intellij-dependencies") }
 }
 
-val lsp: Configuration by configurations.creating
```

Comment:
Most of the stuff below here needs to be evaluated for migration into the new build script, but that should happen very carefully as much of what needed to be explicit in Gradle 1.x is no longer needed explicitly in Gradle 2.x. I think the focus should likely be on bringing over exactly what's needed for the plugin release process since the build/test/run tasks already work well in the current form.